src/java/install.sh

```diff
@@ -206,15 +206,15 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-    # Blank will install latest stable version SDKMAN has
-    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "lts" ] || [ "${requested_version}" = "default" ]; then
+     # Blank will install latest stable version SDKMAN has
+    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "default" ]; then
```

Comment:
Still, are we assuming `latest` to use the `tem` distro? Shouldn't it be the latest version of the given distro?

Reply:
@justinyoo  Why do you think it's always using `tem` ? 

It's using `tem` only if `jdkDistro` is `ms` and version is 8 or 18. @gauravsaini04 Can we check is MS started supporting those two versions as well?

Reply:
no `ms` haven't yet started supporting these 2 versions yet i.e. ones starting with 8 or 18..